raja-perf/ats4/configs/auxiliary_software_files/compilers.yaml

```diff
@@ -0,0 +1,30 @@
+compilers:
+- compiler:
+    spec: clang@16.0.0-rocm5.5.1
+    paths:
+      cc: /opt/cray/pe/craype/2.7.21/bin/cc
+      cxx: /opt/cray/pe/craype/2.7.21/bin/CC
+      f77: /opt/cray/pe/craype/2.7.21/bin/ftn
+      fc: /opt/cray/pe/craype/2.7.21/bin/ftn
+    flags: # TODO: Fix spack concretization bug
```

Comment:
What's the concretization bug here?  What is needed to fix it?

Reply:
I think it is a spack issue https://github.com/spack/spack/issues/39671